claude_scripts/auto_correction_detector.py

```
@@ -0,0 +1,337 @@
+        if not self.memory_file.exists():
+            return []
+
+        try:
+            with open(self.memory_file) as f:
+                data = json.load(f)
+
+            return [
+                c for c in data.get("corrections", []) if not c.get("processed", False)
+            ]
+
+        except:
```

Ruff (0.14.4)

Do not use bare `except`

(E722)